### PR TITLE
#667 Use the network manager provided by QGIS and move it to the file…

### PR DESCRIPTION
**Pull request by Gustry**
#667 Use the network manager provided by QGIS and move it to the file_downloader file

### CODE REVIEW

File: safe/utilities/file_downloader.py [hunk 1]
```diff
@@ -35,12 +35,9 @@
 
 class FileDownloader(object):
     """The blueprint for downloading file from url."""
-    def __init__(self, manager, url, output_path, progress_dialog=None):
+    def __init__(self, url, output_path, progress_dialog=None):
         """Constructor of the class.
```
Comment (timlinux) [inline safe/utilities/file_downloader.py]: .. versionchanged:: 3.3 removed manager parameter.

Comment (Gustry) [inline safe/utilities/file_downloader.py]: Done

**Comment by timlinux** [comment]
Thanks will you go ahead and merge? By the way did you check which version of QGIS this was introduced? Originally we were supporting back to 2.0. Now our officially supported version is the LTR. If we are going to move the requirement up, we need to edit metadata.txt to indicate the new minimum version.

**Comment by Gustry** [comment]
I'm checking in QGIS 2.8.4, but it should be fine as QgsNetworkManager is older than 2.8 in the QGIS repository.

**Comment by Gustry** [comment]
Tested on QGIS 2.8.4, it's ok.
I'm going to merge even if I don't have any proxy. But it should work as we are going to use the QGIS instance.

**Pull request merged**